crates/store/re_log_encoding/src/encoder.rs

```diff
@@ -48,7 +48,66 @@ pub fn encode_to_bytes<'a>(
 
 // ----------------------------------------------------------------------------
 
+/// An [`Encoder`] that can be dropped.
```

Comment:
I mean... the other one can also be dropped :) Maybe "that closes on drop"

Reply:
:grimacing: